examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/with-build-defs.py

```diff
@@ -21,18 +22,17 @@ class ShellScriptSchema(ResolvableSchema):
     asset_specs: Sequence[AssetSpecSchema]
 
 
+def resolve_asset_specs(
```

Comment:
seems like we should be able to generate this one automatically (can do so in follow up)

Reply:
We actually do generate this automatically, but I wanted to make the type conversion stuff explicit in the documentation. Might end up reworking this doc to let that happen automatically though